pkg/mount/mount.sh

```diff
@@ -10,6 +17,24 @@ mkdir -p "$MOUNTPOINT"
 
 mount_drive()
 {
+	# if explicit device identifier was given
+	if [ -n "$DEV_ID" ]
+	then
+		if [ -n "$(echo $DEV_ID | grep -E '^(LABEL|UUID)=')" ]
+		then
+			# identifier is LABEL or UUID
+			DEV=$(findfs $DEV_ID)
+			mount "$DEV" "$MOUNTPOINT" && return
+		elif [ -b "$DEV_ID" ] ; then
+			# identifier is a block-device
+			DEV="$DEV_ID"
+			mount "$DEV" "$MOUNTPOINT" && return
+		else
+			echo "Warning: Invalid Device identifier provided: $DEV_ID"
```

Comment:
This won't return based on the current feedback.  Autodetection of mounting will not be attempted if a device is specified but cannot be found (or mounting it fails.) In that case this message will be displayed, then the lower "WARNING: Failed to mount a persistent volume (is there one?)" message will be displayed.